File: pgm/ABC_Python/ABC_168_A.py
def main():
    N = input()
    N = int(N[len(N) - 1])
    ans = ""
    if N in {2, 4, 5, 7, 9}:
        ans = "hon"
    elif N in {0, 1, 6, 8}:
        ans = "pon"
    else:
        ans = "bon"
    print(ans)
    return


main()

File: pgm/ABC_JavaScript/ABC_168_A.js
"use strict";
function main(input) {
    let N = parseInt(input.replace("\n", "")) % 10;
    if ([2, 4, 5, 7, 9].indexOf(N) !== -1) {
        console.log("hon");
    } else if ([0, 1, 6, 8].indexOf(N) !== -1) {
        console.log("pon");
    } else {
        console.log("bon");
    }
}
main(require('fs').readFileSync('/dev/stdin', 'utf8'));